main.go

```diff
@@ -132,15 +141,29 @@ TimeoutLoop:
 			}
 		}
 
-		networksPruneReport, err := cli.NetworksPrune(context.Background(), args)
-		for _, networkID := range networksPruneReport.NetworksDeleted {
-			deletedNetworks[networkID] = true
-		}
+		try.Do(func(attempt int) (bool, error) {
```

Comment:
This is a completely 3rd party library, or?
So basically just applied [this pattern](https://github.com/matryer/try#delay-between-retries), correct?